sdk/openai/openai/test/public/utils/recordedClient.ts

```diff
@@ -24,6 +24,13 @@ const envSetupForPlayback: { [k: string]: string } = {
 
 const recorderStartOptions: RecorderStartOptions = {
   envSetupForPlayback,
+  sanitizerOptions: {
+    generalSanitizers: [{
+      regex: true,
+      target: `\\.png?[^\"]+`,
+      value: ".png?sanitized"
+    }]
```

Comment:
I am curious, why is this needed? Are you treating the URL as a secret that should be sanitized?

Reply:
There's a SAS token at the end of the URL, so this sanitizer is just getting rid of that.

Reply:
Ah cool, good to know!